tests/e2e/tests/simple/simple1_test.go

```diff
@@ -0,0 +1,198 @@
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
+Simple test - first time:
+source istio.VERSION
+bazel run //tests/e2e/tests/simple:go_default_test -- -alsologtostderr -test.v -v 2 \
+    -test.run TestSimple1 --skip_cleanup --auth_enable --namespace=e2e
+After which to Retest:
+bazel run //tests/e2e/tests/simple:go_default_test -- -alsologtostderr -test.v -v 2 \
+    -test.run TestSimple1 --skip_setup --skip_cleanup --auth_enable --namespace=e2e
+*/
+
+package simple
+
+import (
+	"flag"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/golang/glog"
+
+	"istio.io/istio/devel/fortio"
+	"istio.io/istio/tests/e2e/framework"
+	"istio.io/istio/tests/e2e/util"
+)
+
+const (
+	servicesYaml    = "tests/e2e/tests/simple/servicesToBeInjected.yaml"
+	nonInjectedYaml = "tests/e2e/tests/simple/servicesNotInjected.yaml"
+)
+
+type testConfig struct {
+	*framework.CommonConfig
+}
+
+var (
+	tc *testConfig
+)
+
+func TestMain(m *testing.M) {
+	flag.Parse()
+	check(framework.InitGlog(), "cannot setup glog")
+	check(setTestConfig(), "could not create TestConfig")
+	os.Exit(tc.RunTest(m))
+}
+
+func TestSimpleIngress(t *testing.T) {
+	// Tests the rewrite/dropping of the /fortio/ prefix as fortio only replies
+	// with "echo debug server ..." on the /debug uri.
+	url := "http://" + tc.Kube.Ingress + "/fortio/debug"
+	glog.Infof("Fetching '%s'", url)
+	attempts := 7 // should not take more than 70s to be live...
+	for i := 1; i <= attempts; i++ {
+		if i > 1 {
+			time.Sleep(10 * time.Second) // wait between retries
+		}
+		resp, err := http.Get(url)
+		if err != nil {
+			glog.Warningf("Attempt %d : http.Get error %v", i, err)
+			continue
+		}
+
+		body, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			glog.Warningf("Attempt %d : ReadAll error %v", i, err)
+			continue
+		}
+		_ = resp.Body.Close()
+		bodyStr := string(body)
+		glog.Infof("Attempt %d: reply is\n%s\n---END--", i, bodyStr)
+		needle := "echo debug server on echosrv"
+		if !strings.Contains(bodyStr, needle) {
+			glog.Warningf("Not finding expected %s in %s", needle, fortio.DebugSummary(body, 128))
+			continue
+		}
+		return // success
+	}
+	t.Errorf("Unable to find expected output after %d attempts", attempts)
+}
+
+func TestSvc2Svc(t *testing.T) {
+	ns := tc.Kube.Namespace
+	// Get the 2 pods
+	podList, err := getPodList(ns, "app=echosrv")
+	if err != nil {
+		t.Fatalf("kubectl failure to get pods %v", err)
+	}
+	if len(podList) != 2 {
+		t.Fatalf("Unexpected to get %d pods when expecting 2. got %v", len(podList), podList)
+	}
+	// call into the service from each of the pods
+	for _, pod := range podList {
+		glog.Infof("From pod \"%s\"", pod)
```

Comment:
why `glog.Infof` and not `t.Logf` ?

Reply:
mixer_test and others seem to be using glog.* - I think we may have some capture / log parsing code that uses glog format, not sure. good question for 0.3 testing fw